docs/_posts/examples/3400-01-24-custom-marker-icons.html

```diff
@@ -0,0 +1,93 @@
+---
+layout: example
+category: example
+title: Add custom icons with Markers
+description: 'Use Markers to add custom icons to your map.'
+---
+<div id='map'></div>
+
+<script>
+var geojson = {
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "name": "circle",
+                "icon": {
+                    "iconUrl": "/mapbox-gl-js/docs/assets/circle.png",
```

Comment:
What do you think about eliminating the double occurrence of `icon` in the property access chain?

```js
feature.properties.iconUrl
```